functions/int_select.h

```diff
@@ -0,0 +1,36 @@
+#ifndef FUNCTIONS_INT_SELECT_H
+#define FUNCTIONS_INT_SELECT_H
+
+// Usage: IntSelect<SELECTION, Int1, Int2...>
+// SELECTION: FUNCTION
+// Returns SELECTION of N
```

Comment:
This description still needs work.
An example could work, like say, if SELECTION is 0, the first integer will be returned, if SELECTION is 1, the second, and so forth.

Reply:
updated